raiden/settings.py

```diff
@@ -45,8 +46,8 @@
 ORACLE_BLOCKNUMBER_DRIFT_TOLERANCE = 3
 ETHERSCAN_API = "https://{network}.etherscan.io/api?module=proxy&action={action}"
 
-PRODUCTION_CONTRACT_VERSION = "0.18.0"
-DEVELOPMENT_CONTRACT_VERSION = "0.18.0"
+PRODUCTION_CONTRACT_VERSION = "0.19.0"
```

Comment:
I'd do the same for `PRODUCTION_CONTRACT_VERSION`.

Reply:
Done. We can set it back to a fixed version on release, if that's necessary.